.gitattributes

```diff
@@ -1,11 +1,11 @@
 #
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License
+# , Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License.
+# You may obtain a copy of the License at
 #
 #   http://www.apache.org/licenses/LICENSE-2.0
 #
```

Comment:
Is it because you only modified one part of the license and not the second half?

Reply:
I am not getting it , I made the license header uniform and in all files i have used the same format , so how to fix this problem ?

Reply:
Check it now i fixed that problem